CHANGELOG.md

```diff
@@ -7,6 +7,15 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## [Unreleased]
 
+### Added
+
+- Provide default encoding aliases (#416)
+- Add `get_encoding_by_alias` method to retrieve an encoding by alias
```

Comment:
No need to specify this kind of changes (new methods/classes/..., any code-only changes) in the CHANGELOG, it does not matter at all to scraper users (scrapers are not a library with an API which can be reused by users, at least nobody does it AFAIK). We specify only behavioral / functional changes in scraper CHANGELOG

Reply:
Sorry, I meant only that `Add get_encoding_by_alias method to retrieve an encoding by alias` change is not necessary. Other are required.